pom.xml

```diff
@@ -0,0 +1,285 @@
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-source-plugin</artifactId>
+				<version>3.2.1</version>
+				<executions>
+					<execution>
+						<id>attach-sources</id>
+						<phase>prepare-package</phase>
+						<goals>
+							<goal>jar-no-fork</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.glassfish.copyright</groupId>
+				<artifactId>glassfish-copyright-maven-plugin</artifactId>
+				<version>2.4</version>
+				<executions>
+					<execution>
+						<?m2e ignore?> <!-- Eclipse doesn't seem to like this plugin, it doesn't need to be executed by Eclipse so ignore it -->
+						<id>Check-CopyRight-Notice</id>
+						<goals>
+							<goal>copyright</goal><!-- Checks for Copyright notices -->
+							<goal>check</goal><!-- Fails the build -->
+						</goals>
+						<phase>process-sources</phase>
+					</execution>
+				</executions>
+				<configuration>
+					<excludeFile>${project.basedir}/copyrightExcludeFile.txt</excludeFile>
+					<exclude>
+						DataSourceXAResourceRecoveryHelper.java,NarayanaRecoveryManager.java,FilenameUtils.java,PasswordHash.java
+					</exclude>
+					<ignoreYear>true</ignoreYear>
+					<useDash>true</useDash>
+					<templateFile>${project.basedir}/copyrightTemplateFile.txt</templateFile>
+				</configuration>
+			</plugin>
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.22.2</version>
+				<configuration>
+					<redirectTestOutputToFile>true</redirectTestOutputToFile>
+					<reuseForks>true</reuseForks>
+					<includes>
+						<include>**/*.java</include>
+					</includes>
+					<excludes>
+						<exclude>it.**</exclude>
+					</excludes>
+					<argLine>@{argLine} -Dfile.encoding=UTF-8</argLine>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-deploy-plugin</artifactId>
+				<version>2.7</version>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-release-plugin</artifactId>
+				<version>2.5.3</version>
+				<configuration>
+					<tagNameFormat>v@{project.version}</tagNameFormat>
+					<autoVersionSubmodules>true</autoVersionSubmodules>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-site-plugin</artifactId>
+				<version>3.9.1</version>
+			</plugin>
+			<plugin>
+				<groupId>org.jacoco</groupId>
+				<artifactId>jacoco-maven-plugin</artifactId>
+				<version>0.8.6</version>
+				<executions>
+					<execution>
+						<id>prepare-jacoco</id>
+						<phase>initialize</phase>
+						<goals>
+							<goal>prepare-agent</goal>
+						</goals>
+					</execution>
+					<execution>
+						<id>generate-jacoco-report</id>
+						<phase>test</phase>
+						<goals>
+							<goal>report</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
```

Comment:
De Jacoco plugin is handig voor coverage reports, maar die is niet nodig voor lokale builds. Bij het FF! project heb ik hem daarom verplaatst naar een profiel omdat het toch een paar minuten build tijd scheelt.

Reply:
Gedaan. Ik heb gecontroleerd dat als ik het codecoverage profiel gebruik, dat ik dan de file `target/jacoco.exec` eruit krijg. Dit is geen tekst file, dus ik kon hem verder niet bekijken. Op StackOverflow schrijven ze dat Eclipse en IntelliJ deze file wel kunnen lezen.

Reply:
Als je het codecoverage profiel niet opgeeft, dan wordt jacoco inderdaad niet gebruikt. Ik heb getest dat ik na een Maven Update nog steeds code coverage kan doen in Eclipse.